sql/setup.sql

```diff
@@ -1,2 +1,135 @@
 -- Use this file to define your SQL tables
 -- The SQL in this file will be executed when you run `npm run setup-db`
+DROP TABLE IF EXISTS flowers CASCADE;
+DROP TABLE IF EXISTS planets CASCADE;
+DROP TABLE IF EXISTS fruits CASCADE;
+DROP TABLE IF EXISTS animals CASCADE;
+DROP TABLE IF EXISTS gasses CASCADE;
+
+
+
+-----------Flowers Table----------
+
+CREATE TABLE flowers (
+  id BIGINT GENERATED ALWAYS AS IDENTITY PRIMARY KEY,
+  common_name VARCHAR,
+  color VARCHAR,
+  num_petals INT
+);
+
+INSERT INTO flowers (
+  common_name,
+  color,
+  num_petals
+)
+
+VALUES 
+('checker bloom', 'pink', 5),
```

Comment:
FYI, there is an Array data type in Postgres if you wanted to store the color data as an array. 
https://www.postgresql.org/docs/current/arrays.html